lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -102,6 +102,13 @@ div.course-wrapper {
     @extend .content;
     padding: ($baseline*2);
     line-height: 1.6;
+
+    .xblock {
```

Comment:
@aleffert, sorry for the confusion. No need to integrate right now (since the LMS-based views aren't ready to use the UX Pattern Library styling due to really poorly architected/scoped CSS). My other PR was merely making sure that we have these utilities in general for UI consuming Pattern Library and for the future when the LMS is one of those.

Hope that helps make sense of things.